src/lib/FileTree.svelte

```diff
@@ -23,23 +35,32 @@
         {#each fileTree as file}
             {#if file.children}
                 <li>
-                    <span
-                        role="button"
-                        tabindex={0}
-                        on:click|preventDefault={() => changeViewDirectory(file)}
+                    {#if file.opened}
+                        <button class="expand-btn" on:click={() => file.opened = !file.opened}>
+                            <ChevronDown size="16" color="gray"/>
+                        </button>
+                    {:else}
+                        <button class="expand-btn" on:click={() => file.opened = !file.opened}>
+                            <ChevronRight size="16" color="gray"/>
+                        </button>
+                    {/if}
+                    <button class="expand-btn"
+                        on:dblclick|preventDefault={() => changeViewDirectory(file)}
```

Comment:
Dobbelklikk?
Testet lokalt hvordan det er. Tror vi trenger noe litt mer intuitivt med valg av objekt, det at vi må dobbeltklikke mapper virker ikke opplagt for min del.